.github/workflows/codespell.yml

```diff
@@ -9,10 +9,19 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v39.0.0
+        with:
+          # Ignore all other languages except English
+          files_ignore: |
+            pages.*/*/*
+            package-lock.json
```

Comment:
> the false detection with annotations being made in translation PRs on non-English pages

Why does codespell check non-English pages if they are ignored here?

Reply:
> > the false detection with annotations being made in translation PRs on non-English pages
> 
> Why does codespell check non-English pages if they are ignored here?

It isn't codespell which checks non-English page. It is done by the changed files action which runs on PR which shows suggestion when similar typeface to English is present. 

We have tried multiple methods till date but was unsuccessful in making annotations not show up in unrelated PRs (translations), that's why I removed this action initially but it made codespell unusable as it now parses the whole directory and it's execution get's completed after it detects 15-20 errors/warnings. [These aren't grammatical errors, these annotations show up in places where we use mneumonics, etc]

Reply:
The `files_ignore` option does not seem to do anything; changed-files still detects translations and codespell is clearly being ran:

https://github.com/tldr-pages/tldr/pull/10702/files - invalid suggestions are shown

https://github.com/tldr-pages/tldr/actions/runs/6138227846/job/16654903459#step:5:9
`Running codespell on 'pages.fr/common/csh.md' with the following options...`

Changed-files should not return translation pages with `files_ignore` set.

Reply:
> The `files_ignore` option does not seem to do anything; changed-files still detects translations and codespell is clearly being ran:
> 
> https://github.com/tldr-pages/tldr/pull/10702/files - invalid suggestions are shown
> 
> https://github.com/tldr-pages/tldr/actions/runs/6138227846/job/16654903459#step:5:9
> `Running codespell on 'pages.fr/common/csh.md' with the following options...`
> 
> Changed-files should not return translation pages with `files_ignore` set.

Exactly. But it doesn't block merge on those PRs so I guess it is fine to ignore the annotations (warnings) until we find a fix for the files_ignore part.